src/lib.rs

```diff
@@ -307,16 +307,39 @@ pub struct HwCode {
     pub version: u16,
 }
 
+/// Determine bootstrap address based on hardware code information
+impl From<HwCode> for BootstrapAddr {
+    fn from(hwcode: HwCode) -> Self {
+        match hwcode.code {
+            0x8189 => 0x2001000,
+            _ => 0x201000,
+        }
+    }
+}
+
+type BootstrapAddr = u32;
```

Comment:
We talk about DA (download agent) both in documentation and in the functiosn (send_da/jump_da).  So would probably be better for this to follow that jargon (fwiw it's MTK jargon)

Reply:
Ack, fixed.